www/cordova/chromecast.js

```diff
@@ -47,11 +47,12 @@
         }
     }
 
-    if (MainActivity.supportsPlayStore()) {
-        // Use native chromecast support
-        document.addEventListener('chromecastloaded', onChromecastLoaded);
+    //if (MainActivity.supportsPlayStore()) {
```

Comment:
Since this method was removed from MainActivity.java, I think this should be removed completely.